python/ray/train/BUILD

```diff
@@ -25,10 +21,10 @@ doctest(
     files = [
         "tensorflow/tensorflow_trainer.py",
         "huggingface/transformers/transformers_trainer.py",
-        "accelerate/accelerate_torch_trainer.py",
-        "accelerate/accelerate_torch_trainer_no_raydata.py",
-        "deepspeed/deepspeed_torch_trainer.py",
-        "deepspeed/deepspeed_torch_trainer_no_raydata.py",
+        "examples/accelerate/accelerate_torch_trainer.py",
```

Comment:
should this be defined in the `py_test` examples section below?

Reply:
Done